cookbook/core/extend_flyte/container_interface.py

```diff
@@ -0,0 +1,76 @@
+"""
+.. _core-extend-flyte-container-interface:
+
+Container Interface
+-------------------
+The Flyte typically interacts with containers in the course of its task execution (since most tasks are container
+tasks). If you are extending Flyte with a new container task type, it's worth it to understand how the platform
```

Comment:
I dont follow lines 7/8. i think this should be reworded. Instead of extending, we can just tell how the interaction happens